test/types/string/thomasvandoren/int-to-c_string.bad

```diff
@@ -1,4 +1,3 @@
 5 : int(64)
-5.0 : c_string_copy
-5 : c_string_copy
+5.0 : c_string
```

Comment:
Update the .good file for this test too?

Reply:
D'oh, thanks!